packages/react-components/react-rating-preview/stories/Rating/RatingShape.stories.tsx

```diff
@@ -25,7 +25,7 @@ Shape.parameters = {
   docs: {
```

Comment:
(Can't comment on unchanged lines):
These should both include `step={0.5}`, otherwise it might seem like the half-step is only supported by circles or something.